apps/android/README.md

```diff
@@ -9,6 +9,7 @@ Prereqs:
 - FluentUI Tester is built uses react-native-test app, so install its [prereqs](https://github.com/microsoft/react-native-test-app#react-native-test-app) and follow instructions depending on whether you're developing on Windows or macOS.
 - Install [Android Studio](https://developer.android.com/studio)
 - Make sure you have setup an [Android Virtual Device](https://developer.android.com/studio/run/managing-avds)
+  - M1 Macs: Current best emulator is to still download from Github, [version 3](https://github.com/google/android-emulator-m1-preview)
```

Comment:
nit instead of m1 should we say apple silicon incase there will be m2 chips etc in the future?

Reply:
applies for other documentation as well

Reply:
I would also put a date. Something like "As of July 2021, if you are using an Apple Silicon Mac, ...